grails-app/views/order/_orderInvoices.gsp

```diff
@@ -26,7 +26,7 @@
                                 ${invoiceItem?.description}
                             </g:if>
                             <g:else>
-                                <format:displayName product="${invoiceItem?.product}" productSupplier="${invoiceItem?.orderItem?.productSupplier}" showTooltip="${true}" />
+                                <format:displayName product="${invoiceItem?.product}" productSupplier="${invoiceItem?.shipmentItem?.orderItem?.productSupplier}" showTooltip="${true}" />
```

Comment:
What do you think about adding a getter that would allow us to hide the whole call chain?